addon/components/object-list-view.js

```diff
@@ -572,10 +575,12 @@ export default FlexberryBaseComponent.extend(FlexberryLookupCompatibleComponentM
    * @type Object
    * @param {Boolean} [canBeDeleted=true] The row can be deleted.
    * @param {Boolean} [canBeSelected=true] The row can be selected via checkbox.
+   * @param {String} [customClass=''] The row can has a custon css classes.
```

Comment:
1. Typo: custom
2. It's better to reformulate comment. For example "Custom css classes for the row".